src/aiai_eval/task.py

```diff
@@ -641,6 +664,55 @@ def _adjust_label_ids(
         # TODO: Only a placeholder for now
         return model
 
+    def _prepare_carbon_tracker(
+        self,
+    ) -> Union[EmissionsTracker, OfflineEmissionsTracker]:
+        """Prepares a carbon emissions tracker.
+
+        Returns:
+            EmissionsTracker or OfflineEmissionsTracker:
+                A carbon emissions tracker. OfflineEmissionsTracker is returned if no internet
+                connection is available.
+        """
+        tracker_name = self.task_config.dataset_name
+        if self.evaluation_config.verbose:
+            log_level = "info"
+        else:
+            log_level = "error"
+
+        if internet_connection_available():
+            carbon_tracker = EmissionsTracker(
+                project_name=tracker_name,
+                measure_power_secs=5,
```

Comment:
What does this hyperparameter do, and should a user be able to tweak it?

Reply:
Its how often the power usage is checked in seconds. I think it should not be tweakable as it could be missused to obtain an artificially low emission. So i kept it hardcoded for better comparison between models.